services/api/src/Handlers.ts

```diff
@@ -155,25 +155,21 @@ export function search(db: Database, req: express.Request, res: express.Response
     players: body.players,
     requirespenpaper: body.requirespenpaper,
     text: body.text,
+    showPrivate: body.showPrivate,
   };
 
-  if (params.partition === PUBLIC_PARTITION && res.locals.id) {
-    // Also search for private quests by this user
-    const privateParams = {...params, owner: res.locals.id, partition: PRIVATE_PARTITION};
-    return Promise.all([
-      doSearch(db, res.locals.id, params),
-      doSearch(db, res.locals.id, privateParams),
-    ]).then((results) => {
-      res.status((results[0].error) ? 500 : 200).end(JSON.stringify({
-        error: results[0].error,
-        hasMore: results[0].hasMore,
-        quests: [...results[1].quests, ...results[0].quests],
-      }));
-    });
-  }
-
   return doSearch(db, res.locals.id, params).then((result) => {
-    res.status((result.error) ? 500 : 200).end(JSON.stringify(result));
+    const sortedQuests: Quest[] = result.quests.sort((a: Quest, b: Quest) => {
+      const questAPartition = a.partition.toLowerCase();
+      const questBPartition = b.partition.toLowerCase();
+      if (questAPartition <= questBPartition) {
```

Comment:
This can be streamlined to `if (a.partition.toLowerCase() <= b.partition.toLowerCase()) {`; it's only a few characters longer but saves two lines, and those variables aren't re-used :)